src/main/java/uk/gov/companieshouse/company/metrics/consumer/PscStatementsStreamConsumer.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.companieshouse.company.metrics.consumer;
+
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.kafka.annotation.RetryableTopic;
+import org.springframework.kafka.retrytopic.FixedDelayStrategy;
+import org.springframework.kafka.support.KafkaHeaders;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.handler.annotation.Header;
+import org.springframework.retry.annotation.Backoff;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.company.metrics.exception.NonRetryableErrorException;
+import uk.gov.companieshouse.company.metrics.processor.MetricsRouter;
+import uk.gov.companieshouse.company.metrics.type.ResourceChange;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.stream.ResourceChangedData;
+
+@Component
+public class PscStatementsStreamConsumer {
```

Comment:
Is it worth having a more generic naming convention for our PSCs route? Just thinking we can probably re-use the consumer, router and client for PSCs

Reply:
Sure, I'll rename it to just PSC